.github/approve_config.yml

```diff
@@ -2,16 +2,25 @@ approvals:
   # check will fail if there is no approval
   minimum: 1
   groups:
-    - name: GridAI
-      minimum: 1
+    - name: 'PyTorch Lightning'
+      minimum: 0
       from:
         - awaelchli
         - Borda
         - carmocca
-        - edenlightning
         - ethanwharris
         - kaushikb11
         - krshrimali
+        - otaj
         - rohitgr7
         - tchaton
         - williamFalcon
+    - name: 'Lightning Apps'
```

Comment:
How does this link to the source folder?

Reply:
It doesn't. But I don't think it's necessary because this is just the employee approval.